conbench/app/benchmarks.py

```diff
@@ -229,6 +264,12 @@ def show_trends_for_benchmark(bname: TBenchmarkName) -> str:
         caseid, ctxid, hwid = t3
         results = bmrt_cache["by_4t_list"][(bname, caseid, ctxid, hwid)]
 
+        # dfts = bmrt_cache["by_4t_df"][(bname, caseid, ctxid, hwid)]
+        # print()
+        # print()
+        # print((bname, caseid, ctxid, hwid))
+        # print(dfts.to_csv(na_rep="NaN", float_format=numstr8))
```

Comment:
this is the method via which I generated the test scenario time series CSV files